### ISSUE TITLE
Possible question to answer 

### ISSUE THREAD

**Issue by Shan-non** [UBC-MDS/maternal_health_classification#2]
What feature is the strongest predictor of maternal health risk?
- split data into test and train sets
- create a pipeline with StandardScaler, and a logistic regression model. 
- choose best hyperparameters based on randomized search
- using best hyperparameters fit the model on the training set
- score using test set to see the model accuracy. 
- view the coefficients the model has identified for the features and from there identify the strongest feature predictor of the target (feature with the largest absolute value).

Limitations:
- if our model scores terribly on our test set is choosing a best feature predictor meaningful?
- in the case of all features being similar our question may be impossible to answer with a logistic regression model

**Comment by Shan-non** [comment]
Our question will be about predicting maternal health risk from the features included in our dataset: age, systolic and diastolic blood pressure, heart rate, blood sugar, and body temperature.
We will compare a logistic regression model (and in this area see if which (if any) feature is the best predictor in this model), and a Naive bayes model when predicting on the maternal health risk target.

**Comment by hovhannisyannelli** [comment]
Have decided the suggested question and answer
